Add docs for jslib http-instrumentation-tempo (#1555)
grafana/k6-docs: docs/sources/next/javascript-api/jslib/http-instrumentation-tempo/instrumenthttp.md

## Parameters

| Name      | Type                                                                                                 | Description                                                                                                                                                    |
| :-------- | :--------------------------------------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| `options` | [`Options`](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/jslib/http-instrumentation-tempo/options) | Configures the tracing behavior with the provided [`Options`](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/jslib/http-instrumentation-tempo/options) object. |

## Example

> This example demonstrates how calling the `instrumentHTTP` function in the init context of a script once ensures that all requests made by the HTTP module from that point forward will have a trace context header attached.

Reviewer: ```suggestion
This example demonstrates how to call the `instrumentHTTP` function in the script's init context. From that point forward, all the requests made by the HTTP module have a trace context header attached.
```
